views/model.njk

```diff
@@ -38,7 +38,7 @@
     <div class="content box">
       <pre style="border: 0; background-color: transparent;">
       <code class="js">
-      {% for decl in modelFile.getAllDeclarations() %}import {{decl.getFullyQualifiedName()}} from {{serverRoot}}{{filePath}}.cto
```

Comment:
I don't think the import should be quoted in CTO syntax.

Reply:
If quotes are removed, the part of the statement after "https:" is displayed as a comment, due to syntax highlighting (using highlight.js). It probably detects the code as javascript

![Screenshot (99)](https://github.com/accordproject/models/assets/112375644/bd344152-7f78-4b3c-a0ae-092c133be141)

As a workaround, I ended up skipping syntax-highlighting for this code block altogether. So this is how it looks like now-

![Screenshot (98)](https://github.com/accordproject/models/assets/112375644/cb070626-0fdc-4129-ae08-a8ef451f4205)